telemetry/specs/INT.mdk

```diff
@@ -298,18 +298,64 @@ exhausting header space in the case of a forwarding loop or any other anomalies,
 it is strongly recommended to limit the number of total INT metadata fields
 added by devices.
 
+### MTU Settings
+
 As with any modification that potentially changes the packet size, this
-on-the-fly allocation may “grow” a packet beyond the original MTU, resulting
-in fragmentation. It is advisable to plan network/NIC/MTU settings accordingly
-to leave headroom for the additional headers. 
+on-the-fly allocation may “grow” the packet size, potentially causing it to
+exceed the MTU on an INT switch egress link.
+
+This may be addressed in the following ways -
+
+* It is recommended that the MTU of links between INT sources and sinks is
+configured to be higher than the MTU of preceding link MTUs (server/VM NIC MTUs)
+by an appropriate amount. Configuring an MTU differential of
+(Instruction Count\*4\*INT Hop Count + 8), based on conservative values of
+total number of INT hops and instruction count being used, will prevent egress
+MTU being exceeded due to INT metadata insertion at INT hops.
+
+* An INT source/transit switch may optionally aid in dynamic discovery of Path MTU
+along flows being monitored by INT by transmitting ICMP message as per Path MTU
+Discovery mechanisms of the corresponding IP protocol
+(RFC 1191 for IPv4, RFC 1981 for IPv6). An INT source or transit switch may report
+a conservative MTU in the ICMP message, accounting for metadata insertion at
+downstream INT hops and assuming uniform egress MTU at downstream hops. This
```

Comment:
Not just downstream hops. All hops including upstream and downstream should be subtracted from the underlying real MTU of the egress link.

Reply:
Agreed, fixed this.